recipes/minys/meta.yaml

```diff
@@ -0,0 +1,40 @@
+{% set version = "1.0" %}
+{% set sha256 = "c26d48e3d1002ee79e256ce1bae050178a24f8c02b10233868cd2a4cf2e30b27" %}
+
+package:
+  name: minys
+  version: {{ version }} 
+
+build:
+  number: 0
+  # requires clang>=4.5
+  # skip: True  # [osx]
+
+source:
+  url: https://github.com/cguyomar/MinYS/releases/download/v{{ version }}/MinYS-v{{ version }}-Source.tar.gz
+  sha256: {{ sha256 }}
+
+requirements:
+  build:
+    - {{ compiler('cxx') }}
+    - make
+  run:
+    - mindthegap
+    - minia
+    - bwa
+    - samtools
+
+test:
+  commands:
+    - MinYS.py -h
+
+about:
+  home: https://github.com/cguyomar/MinYS
+  license: file
+  license_file: LICENSE
+  summary: MinYS allows targeted assembly of bacterial genomes using a reference-guided pipeline.
+
+extra:
+  identifiers:
+    - biotools:minys
```

Comment:
you should only have this if minys is registered at https://bio.tools/
-> you can register it or just remove this identifier block

Reply:
Thanks for noticing.
resolved in last commit